src/widgets/programs/programs-form.css

```diff
@@ -77,3 +77,35 @@
 .buttonStylesBorder {
   border-top: 0.125rem solid var(--omrs-color-interaction);
 }
+
+input[type="date"]:invalid {
```

Comment:
This now applies to all invalid date input types. Should we set this in the `styleguide` instead?

Reply:
Indeed we should. I've submitted a PR for this https://github.com/openmrs/openmrs-esm-styleguide/pull/75